hnn_core/viz.py

```diff
@@ -93,6 +116,9 @@ def plot_dipole(dpl, tmin=None, tmax=None, ax=None, layer='agg', decim=None,
     import matplotlib.pyplot as plt
     from .dipole import Dipole
 
+    # NB units_str is applied to the label, includes parentheses: ' (nAm)'
+    scaling, units = _check_scaling_units(scaling, units)
```

Comment:
```suggestion
```

Reply:
All tests and examples with `postproc=True` would fail...